### PR TITLE
adding gh-pages, hash  instead of browser router

### PR DESCRIPTION
**Pull request by emil-owczarek**
## Click Up Task Link  ⏰ 
[Infrasctructure]: GitHub CD - deploy to GH pages from master
https://app.clickup.com/t/86930671t

## What has been done 😄 
gh-pages dependencies added, browserRouter changed to hashRouter 

Not sure if deployment is going to work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,13 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "homepage": "https://pl-fe-may23-codecreators.github.io",
```
Comment (klaudia-klis) [inline package.json]: I'm gettin 404 error on this page, do you know why and how to fix that? Maybe the homepage should be https://pl-fe-may23-codecreators.github.io/product-catalog/ ? I'm not sure...

Comment (emil-owczarek) [reply package.json]: Not sure either. It seems to me that first I need to merge it with main, and then it will be deployed to a good website

Comment (emil-owczarek) [reply package.json]: But I'll leave that for Monday's Q&A then

Comment (klaudia-klis) [reply package.json]: I can give you an approve if you want to check your theory :D

**Review by klaudia-klis — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (emil-owczarek) [reply package.json]: it is more of a guess than a theory, but let's try! I changed the name of homepage so I can blame you later.

**Pull request merged**